share/spice/just_delete_me/just_delete_me.js

```diff
@@ -13,7 +13,9 @@
             return Spice.failed('just_delete_me');
         }
         var matchPartial = function(item) {
-            return (item.name.toLowerCase().includes(this) || item.domains.some(domain => domain.includes(this)));
+            return (item.name.toLowerCase().includes(this) || item.domains.some(function(domain) {
+                return domain.includes(this)
```

Comment:
This will not work. this is by default undefined inside an anonymous function, it must be passed as the second argument to the some() or filter() call.